### PR TITLE
Media info

### PR DESCRIPTION
**Pull request by RyanRory**

### CODE REVIEW

File: SessionMessagingKit/Sending & Receiving/Notifications/PushNotificationAPI.swift [hunk 1]
```diff
@@ -28,7 +28,7 @@ public final class PushNotificationAPI : NSObject {
     }
 
     // MARK: - Settings
-    public static let server = "https://live.apns.getsession.org"
```
Comment (mpretty-cyro) [inline SessionMessagingKit/Sending & Receiving/Notifications/PushNotificationAPI.swift:31]: Please revert the notification server change

File: Session/Media Viewing & Editing/MediaInfoVC.swift [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+    }
+    
+    // MARK: - Lifecycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        ViewControllerUtilities.setUpDefaultSessionStyle(
+            for: self,
+            title: "Message Info",
```
Comment (mpretty-cyro) [inline Session/Media Viewing & Editing/MediaInfoVC.swift:103]: Please make this a localised string

**Review by mpretty-cyro — CHANGES_REQUESTED** [review]

**Review by mpretty-cyro — APPROVED** [review]

File: Session/Media Viewing & Editing/MediaInfoVC.swift (hunk 2 above)
Comment (RyanRory) [reply Session/Media Viewing & Editing/MediaInfoVC.swift]: Done

File: SessionMessagingKit/Sending & Receiving/Notifications/PushNotificationAPI.swift (hunk 1 above)
Comment (RyanRory) [reply SessionMessagingKit/Sending & Receiving/Notifications/PushNotificationAPI.swift]: Done.

**Review by mpretty-cyro — APPROVED** [review]

**Pull request merged**